src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BeeWingListener.java

```diff
@@ -0,0 +1,70 @@
+package io.github.thebusybiscuit.slimefun4.implementation.listeners;
+
+import org.bukkit.HeightMap;
+import org.bukkit.Location;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerMoveEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunItems;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.utils.SlimefunUtils;
+import me.mrCookieSlime.Slimefun.api.Slimefun;
+
+/**
+ * This {@link Listener} is responsible for the slow falling effect given to the player
+ * when nearing the ground while using the Bee Wings.
+ *
+ * @author beSnow
+ * 
```

Comment:
@LinoxGH You can pretty much add yourself as an author here below too at this point.